app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoadTaxonomiesService.java

```diff
@@ -82,12 +58,8 @@ private void doTask(Intent intent) {
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .toCompletable()
-                //.doOnSubscribe(d -> view.showLoading())
                  .subscribe(() -> {
                     //view.hideLoading(false);
-                  //   builder.setContentText(getString(R.string.txtLoaded));
-                 //    builder.setOngoing(false);
-                 //    notificationManager.notify(17, builder.build());
                 }, e -> {
                    //  e.printStackTrace();
```

Comment:
If `onComplete` and `onError` is not needed, just call `subscribe()` with no params.

Also as mentioned by @deniger, could you remove the string `txtDownloading` in strings.xml .

Reply:
changes made :)